mobile/screens/SupplyAdd.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import {Button, Image, ScrollView, StyleSheet, View} from 'react-native';
+import {Input} from "react-native-elements";
+import {postSupply} from "../services/SuppliesService";
+
+
+export default class SupplyEdit extends React.Component {
+
+    state = {
+        isShowingText: true,
+        supply: {
+            name: "",
+            state: "",
+            description: ""
+        }
+    };
+
+
+    static navigationOptions = {
+        header: null
+    };
+
+    onPressSave = () => {
+        const {goBack} = this.props.navigation;
+        const newSupply = {
+            name: this.state.supply.name,
+            state: this.state.supply.state,
+            description: this.state.supply.description
+        }
+        postSupply(newSupply)
+            .then(() => this.props.navigation.state.params.onRefresh())
+        goBack()
+    }
+
+    render() {
+        const {supply} = this.state
+        return (
+            <View style={styles.container}>
+                <ScrollView style={styles.container}>
+                    {this.state.supply && <View style={styles.mainView}>
+                        <View style={styles.imageContainer}>
+                            <Image
+                                source={{uri: 'https://via.placeholder.com/150'}}
+                                resizeMode="contain"
+                                fadeDuration={0}
+                                style={styles.image}
+                            />
+                        </View>
+                        <Input inputStyle={styles.optionsTitleText}
+                               onChange={(e) => {
+                                   this.setState({supply: {...supply, name: e.nativeEvent.text}})
+                               }}
+                               value={supply.name}
+                               label={"Nazwa"}
```

Comment:
I think we want to use English as language for our app.

Reply:
task for translations was added.